grafana-operator/grafana-operator
# [WIP] Loki

## Description

Work in progress to revise the CRDs, allow auto generation and start supporting Loki and Promtail types.



## Type of change



## Checklist


## Verification steps

1. deploy a previous version or the operator, Grafana and some dashboards
2. ugprade the operator to this version
3. the dashboards should get reimported, the Grafana CR should get updated to remove the `dashboards` field (which is a map) and replace it with the `knownDashboards` field (which is an array).
